message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -41,9 +41,9 @@
       </entry>
       <entry value="83" name="MAV_CMD_NAV_ALTITUDE_WAIT">
         <description>Mission command to wait for an altitude or downwards vertical speed. This is meant for high altitude balloon launches, allowing the aircraft to be idle until either an altitude is reached or a negative vertical speed is reached (indicating early balloon burst). The wiggle time is how often to wiggle the control surfaces to prevent them seizing up.</description>
-        <param index="1">Altitude (m).</param>
-        <param index="2">Descent speed (m/s).</param>
-        <param index="3">Wiggle Time (s).</param>
+        <param index="1" label="Altitude" units="m">Altitude.</param>
+        <param index="2" label="Descent Speed" units="m/s">Descent speed.</param>
+        <param index="3" label="Wiggle Time" units="s">Wiggle time.</param>
```

Comment:
WTF is "Wiggle time"?!

Yes, I could look it up, but I shouldn't really have to....